doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2725 @@
+     [  9  10  11  12]] 
+  New array:  
+    [101 2 3 4 5 6 7 8 9 10 11 12]
+
+`Read more about flatten here <https://numpy.org/devdocs/reference/generated/numpy.ndarray.flatten.html>`_ and
+`ravel here <https://numpy.org/devdocs/reference/generated/numpy.ravel.html#numpy.ravel>`_.
+
+
+How to access the docstring for more information
+------------------------------------------------
+
+::
+
+  help() ? ??
+
+-----
+
+When it comes to the data science ecosystem, Python and NumPy are built with the
+user in mind. One of the best examples of this is the built-in access to
+documentation. Every object contains the reference to a string, which is known
+as the **docstring**. In most cases, this docstring contains a quick and concise
+summary of the object and how to use it. Python has a built-in ``help()``
+function that can help you access this information. This means that nearly any
+time you need more information, you can use ``help()`` to quickly find the
+information that you need.
+
+For example,
+
+::
+
+  help(max)
+
+Will return
+
+::
+
+  Help on built-in function max in module builtins:
+
+  max(...) max(iterable, *[, default=obj, key=func]) -> value max(arg1, arg2,
+  *args, *[, key=func]) -> value
+      
+      With a single iterable argument, return its biggest item. The default
+      keyword-only argument specifies an object to return if the provided
+      iterable is empty. With two or more arguments, return the largest
+      argument.
+
+<<<<<<< HEAD
+Because access to additional information is so useful, IPython uses the ``?``
+character as a shorthand for accessing this documentation along with other
+relevant information. IPython is a command shell for interactive computing in
+multiple languages.`You can find more information about IPython here
+<https://ipython.org/>`_. 
+=======
+Because access to additional information is so useful, IPython uses the ``?`` character as a shorthand for accessing this documentation along with other relevant information. IPython is a command shell for interactive computing in multiple languages. `You can find more information about IPython here <https://ipython.org/>`_. 
+>>>>>>> e1969d86e2fe034c46b8943a40282007ec561b6f
+
+For example,
+
+::
+
+  max?
+
+Will return:
+
+::
+
+  Docstring: max(iterable, *[, default=obj, key=func]) -> value max(arg1, arg2,
+  *args, *[, key=func]) -> value
+
+  With a single iterable argument, return its biggest item. The default
+  keyword-only argument specifies an object to return if the provided iterable
+  is empty. With two or more arguments, return the largest argument. Type:     
+  builtin_function_or_method
+  
+You can even use this notation for object methods and objects themselves.
+
+Let's say you create this array:
+
+::
+
+  a = np.array([1, 2, 3, 4, 5, 6])
+
+Running
+
+::
+
+  a?
+  
+Will return a lot of useful information.
+
+::
+
+  Type:            ndarray String form:     
+                  [1 2 3 4 5 6] Length:          6
```

Comment:
Line wrap tool interfering again? I think you might need to roll back to before that commit.

Reply:
Line wrap tool was a big issue, but the problems should be resolved now